### ISSUE TITLE
NBCNews Unable to extract bootstrap json

### ISSUE THREAD

**Issue by anonvoy** [ytdl-org/youtube-dl#25871]
## Checklist

- [x] I'm reporting a broken site support
- [x] I've verified that I'm running youtube-dl version **2020.06.16.1**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
abc@Linux:~> youtube-dl -U
youtube-dl is up-to-date (2020.06.16.1)
abc@Linux:~> youtube-dl -v https://www.nbcnews.com/nightly-news-netcast/video/nightly-news-full-broadcast-july-1st-86722117902
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', 'https://www.nbcnews.com/nightly-news-netcast/video/nightly-news-full-broadcast-july-1st-86722117902']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2020.06.16.1
[debug] Python version 3.6.10 (CPython) - Linux-4.12.14-lp151.28.52-default-x86_64-with-glibc2.3.4
[debug] exe versions: ffmpeg 3.4.5, ffprobe 3.4.5
[debug] Proxy map: {}
[NBCNews] 86722117902: Downloading webpage
ERROR: Unable to extract bootstrap json; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 797, in extract_info
    ie_result = ie.extract(url)
  File "/usr/bin/youtube-dl/youtube_dl/extractor/common.py", line 530, in extract
    ie_result = self._real_extract(url)
  File "/usr/bin/youtube-dl/youtube_dl/extractor/nbc.py", line 398, in _real_extract
    'bootstrap json'), video_id, js_to_json)
  File "/usr/bin/youtube-dl/youtube_dl/extractor/common.py", line 1005, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
youtube_dl.utils.RegexNotFoundError: Unable to extract bootstrap json; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

```

## Description

Stopped working ca. 2 days ago, I could still download the June 29 broadcast (https://www.nbcnews.com/nightly-news-netcast/video/nightly-news-full-broadcast-june-29th-86437445994). That is still up, but youtube-dl doesn't download it anymore.

**Comment by gregbert42** [comment]
same.

**Comment by gregbert42** [comment]
I still dont see a solution for this, other than to switch to YouTube downloading. @anonvoy - I've written a python script to download this each night from youtube if you want it. I am an expat so this is how I get my news onto my plex server.

the function is below. just pass "NBC Nightly News' and the first result is usually the news from the day before, which can be directly downloaded via youtube-dl

`def SearchYouTube(TextToSearch):
    import urllib.request
    import urllib.parse
    import re
    print ("YouTube Query: %s" % TextToSearch)
    query_string = urllib.parse.urlencode({"search_query" : TextToSearch})
    html_content = urllib.request.urlopen("http://www.youtube.com/results?" + query_string)
    search_results = re.findall(r'watch\?v=(.{11})', html_content.read().decode())
    print (f'Search results: {search_results}')
    return (f'https://www.youtube.com/watch?v={search_results[0]}')
`

**Comment by anonvoy** [comment]
> I still dont see a solution for this, other than to switch to YouTube downloading. @anonvoy - I've written a python script to download this each night from youtube if you want it. I am an expat so this is how I get my news onto my plex server

Thanks, but I got the periodic Youtube download from the NBCNews channel there covered with Flexget and the Youtube channel's RSS feed. Unfortunately, both the Youtube channel and the NBCNews site sometimes miss episodes, mostly on weekends, and the quality of the Youtube versions is sometimes quite bad.

From the NBCNews site, you can get the URLs of the single .ts segments, then download those segments (ca. 130 to 140 per episode) and finally combine them to a single file. I use various browser extensions and programs for this (Firefox Developer Tools, Flashgot, JDownloader, mkvtoolnix), but it's somewhat time consuming.

**Comment by remitamine** [comment]
fix with the latest version(2020.11.17).

**Comment by gregbert42** [comment]
Hallelujah
